packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/ViewModifier.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams;
+
+import java.util.Collection;
+import java.util.Optional;
+
+/**
+ * All possible modifiers to define the state of a diagram element.
+ *
+ * @author tgiraudet
+ */
+public enum ViewModifier {
+
+    Hidden, Faded, Normal;
+
+    public static final ViewModifier DEFAULT_MODIFIER = ViewModifier.Normal;
```

Comment:
Why should we use `ViewModifier.DEFAULT_MODIFIER` instead of `ViewModifier.NORMAL`? It's not even shorter. This can be removed.